src/exercises/01-Query-Renderer/README.md

```diff
@@ -223,24 +225,99 @@ _TODO: add a gif_
+        }
+      `}
+      // ...
+    />
+```
+
+_TODO: line numbers_
+
+Notice that we add the variable to the top-level `query` as well as the `artist` type we're querying. 
+
+At this point you'll get a type error on the `variables` prop. This is because the query is expecting a property named `artistID` on the `variables` argument.
+
+💻 _Add a hardcoded `artistID` to the `variables` prop:_
+
+```typescript
+    <QueryRenderer<Artist1QueryRendererQuery>
+      // ...
+      variables={{artistID: '1'}}
+      // ...
+    />
+```
+
+_TODO: line numbers_
+
+Save and refresh and you'll see...not much is different. We merely moved the hardcoded value from one place to another.
+
+But! Now that we're passing the `artistID` _into_ the `QueryRenderer`, we can pass whatever we want — like the ID from the current route!
 
 ##### Step 2: Grab value of variable from route 
 
-## TODO:
+We're already grabbing the artistID from the route params. We need to pass it into our query. 
+
+💻 _Replace the hardcoded `artistID` variable with the value from the route parameter:_
+
+```typescript
+    <QueryRenderer<Artist1QueryRendererQuery>
+      // ...
+      // pass the artistID variable we extracted from the route:
+      variables={{artistID}}
+      // ...
+    />
+```
+
+_TODO: line numbers_
+
+Now when we visit [Kehinde Wiley's page](http://localhost:1234/exercise-1/artist/3) we see his info! 🕺💃
+
+![The artist detail screen showing Kehinde's info](TODO)
 
-- See if there are other things we specify in force/eigen when using QueryRenderers that are worth calling out
+## Wrapping up
 
-## Summary
+In this exercise we connected our React component to our GraphQL endpoint with a Relay `QueryRenderer`. (Technically a `react-relay` `QueryRenderer` — [all the React components associated with Relay exist in that package](https://relay.dev/docs/en/architecture-overview#core-modules).)
 
-- Summarize what the query renderer is doing to connect the component to the graphql API
-  - react-relay vs relay
-    - relay doesn't include react components; react-relay does
+When our component first renders, our `QueryRenderer`'s `render` prop gets called, with empty `props`. This paints a `Loading` indicator to the screen.
+
+Meanwhile, Relay makes a network request to our GraphQL endpoint. It passes the `query` and `variables` props from our `QueryRenderer` to the GraphQL server. 
+
+When the server responds, the `QueryRenderer` calls the `render` prop again, this time with `props` loaded from the network response. This renders our component tree, hydrated with data. 
+
+In day-to-day development with Relay, you'll work with QueryRenderers occasionally — typically when you're adding a new page or screen, or when you need a test or storybook to connect a component to GraphQL data. 
 
 ## Recommendations
 
 ### Be careful nesting QueryRenderers
 
+Each QueryRenderer is associated with one request to our GraphQL server. Nesting QueryRenderers therefore has a cascading effect:
```

Comment:
TODO: "Nesting" & "cascading effort" require a bit more explanation here. It's not totally clear what we're doing to cause the problem, or why there _is_ a problem.

```suggestion
Each QueryRenderer is associated with one request to our GraphQL server. As we continue to build out our Artist page, we'll want to add child components for things like their artworks, their bio, etc. What we've learned so far might give us the impression that each of those child components would be associated with its own QueryRenderer. This would cause a problem — the child components' QueryRenderers would make separate network requests, causing performance issues.
The correct approach to this situation is to associate the child components with something called a FragmentContainer — which we'll learn about in the next exercise.
```